rasa/shared/core/domain.py

```diff
@@ -1963,9 +1995,9 @@ def _validate_slot_mappings(domain_slots: Dict[Text, Any]) -> None:
         )
 
     rasa.shared.utils.io.raise_warning(
-        "Slot auto-fill has been removed in 3.0 and replaced by a"
-        " new explicit mechanism to set slots. "
-        "Please refer to the docs to learn more.",
+        f"Slot auto-fill has been removed in 3.0 and replaced with a"
+        f" new explicit mechanism to set slots. "
+        f"Please refer to {DOCS_URL_SLOTS} to learn more.",
```

Comment:
Can we please investigate why the docs url isn't shown in the user warning? This would in my opinion be crucial to fix.

Reply:
I've quickly looked into this - it seems that this [line](https://github.com/RasaHQ/rasa/blob/dc0af420818e263fb4ef97c0d7f1c65e1da83bd1/rasa/shared/utils/io.py#L80) will evaluate to `False` because `kwargs` [here](https://github.com/RasaHQ/rasa/blob/dc0af420818e263fb4ef97c0d7f1c65e1da83bd1/rasa/shared/utils/io.py#L64) is equal to `{'stacklevel': 2}`.